<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>one</class>
 <widget class="QDialog" name="one">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>历史数据</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/1/app.png</normaloff>:/1/app.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>返回主界面</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/4/fanhui1.png</normaloff>:/4/fanhui1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="xieru">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>开始写入</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/4/xierushujuku.png</normaloff>:/4/xierushujuku.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>保存节点一</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/4/baocun.png</normaloff>:/4/baocun.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="qc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>清除节点一</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/1/qingchu.png</normaloff>:/1/qingchu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="save_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>保存节点二</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/4/baocun.png</normaloff>:/4/baocun.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="save_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>保存节点三</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/4/baocun.png</normaloff>:/4/baocun.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="qc_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>清除节点二</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/1/qingchu.png</normaloff>:/1/qingchu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="qc_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>清除节点三</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/1/qingchu.png</normaloff>:/1/qingchu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background: transparent; 
	border: none; 
	color: rgb(211, 215, 207)

}</string>
    </property>
    <property name="text">
     <string>暂停写入</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/1/zantinghuizhi.png</normaloff>:/1/zantinghuizhi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="text1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>335</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget 
{ 
background-color: transparent; 
color: rgb(18, 150, 219);
gridline-color: rgb(18, 150, 219);
}
QHeaderView::section {
color: rgb(18, 150, 219);
border: none; 
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="text2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>100</y>
     <width>335</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget 
{ 
background-color: transparent; 
color: rgb(26, 250, 41);
gridline-color: rgb(26, 250, 41);
}
QHeaderView::section {
background-color: transparent; 
color: rgb(26, 250, 41);
border: none; 
}
</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="text3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>335</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget 
{ 
background-color: transparent; 
color:rgb(244, 234, 42);
gridline-color: rgb(244, 234, 42);
}
QHeaderView::section {
color: rgb(244, 234, 42);
border: none; 
}

</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
